app/cluster_handlers.go

```diff
@@ -34,6 +35,35 @@ func (a *App) clusterInstallPluginHandler(msg *model.ClusterMessage) {
 	a.InstallPluginFromData(model.PluginEventDataFromJson(strings.NewReader(msg.Data)))
 }
 
+func (a *App) clusterPluginEventHandler(msg *model.ClusterMessage) {
+	env := a.GetPluginsEnvironment()
+	if env == nil {
+		return
+	}
+	if msg.Props == nil {
+		mlog.Warn("ClusterMessage.Props for plugin event should not be nil")
+		return
+	}
+	pluginID := msg.Props["PluginID"]
+	eventID := msg.Props["EventID"]
+	if pluginID == "" || eventID == "" {
+		mlog.Warn("Invalid ClusterMessage.Props values for plugin event",
+			mlog.String("PluginID", pluginID), mlog.String("EventID", eventID))
```

Comment:
nit: for logging purposes, I've found it helpful that existing logs use `plugin_id` and don't require me to search for permutations. Worth logging as snake case as such?

Reply:
Sounds good to me. I am all for consistency :)